workloads/kube-burner/common.sh

```diff
@@ -14,11 +14,11 @@ else
   export PROM_TOKEN=$(oc -n openshift-monitoring sa get-token prometheus-k8s)
 fi
 export TOLERATIONS="[{key: role, value: workload, effect: NoSchedule}]"
-export UUID=${UUID:-$(uuidgen)}
+export UUID=${UUID:-$(uuidgen | awk '{print tolower($0)}')}
```

Comment:
AFAIK uuidgen generates a lowercase uuid 🤔

Reply:
I'm not sure if it was lowercase on my mac, but a lot of the other commands I can't run. So I'll remove